apps/mailman/templates/mailman/survey_email_html.py

```diff
@@ -6,63 +6,42 @@ def write_html(email_text, link_html):
-    <img alt="WorkElections" src="https://s3.amazonaws.com/voteworker/workelections.png">
-        <br><br>
-              WorkElections.com Request for Information
-           </td>
-           <td width="20">&nbsp;</td>
-           <td width="20">&nbsp;</td>
-       </tr>
-       <tr bgcolor="#39446f">
-           <td colspan="5" height="40">&nbsp;</td>
-       </tr>   
-       <tr>
-           <td height="20" colspan="5">&nbsp;</td>
-       </tr>
-    <tr>
+    <tbody><tr>
+       <td align="center" style="padding-left:30px; padding-right:30px"><img width="200" alt="WorkElections" src="https://s3.amazonaws.com/voteworker/workelections.png"></td>
+       <td align="center" style="padding-left:30px; padding-right:30px"><img width="240" alt="Fair Elections Center" src="https://www.workelections.com/graphics/meta/FairElectionsCenterLogo.png"></td>
         <td>&nbsp;</td>
-        <td colspan="3" align="left" valign="top" style="color:#666666; font-size: 13px;">
+    </tr>
+    <tr><td colspan="2" style="padding-top:50px; padding-bottom:20px"><hr></td></tr>
+    <tr>
+        <td colspan="2" align="left" valign="top" style="color:#555555; padding-left:30px; padding-right:30px; padding-bottom:20px;">
             
-                <p> <pre>""" + email_text + """ </pre> </p>
+                <p>""" + email_text + """</p>
             
         </td>
-        <td>&nbsp;</td>
     </tr>
-        <tr>
-            <td colspan="5" height="30">&nbsp;</td>
-        </tr>
-        <tr>
-            <td>&nbsp;</td>
-            <td colspan="3">
-                <table border="0" cellpadding="0" cellspacing="0" align="left"
-                    <tr>
-                        <td align="center" valign="center">""" + link_html + """
-                        </td>
-                    </tr>
-                </table>
-            </td>
-            <td>&nbsp;</td>
-        </tr>
-        <tr>
-            <td colspan="5" height="30">&nbsp;</td>
-        </tr>
-    <tr>
-        <td height="20" colspan="5">&nbsp;</td>
-    </tr>
-    <tr>
-        <td height="20" colspan="5">&nbsp;</td>
-    </tr> 
+    <tr><td colspan="4" align="center">""" + link_html + """</td></tr></tbody>
+    <tr><td colspan="4" align="center" style="padding:10px; background-color:#BBBBBB; color:white; font-size:12px">
+    1825 K St. NW  | Suite 450<br>
+    Washington DC 20006<br>
+    (202) 331-0114<br>
+    <a style="color:#A5002F" href="mailto:info@fairelectionscenter.org">info@fairelectionscenter.org</a><br>
+    </td></tr>
 </table>
 </div>
 </body>
 </html>
-"""
\ No newline at end of file
+"""
+
+def write_button(url, label):
+    return '''
+<td style="padding:10px"><div><!--[if mso]>
+  <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="http://" style="height:42px;v-text-anchor:middle;width:130px;" arcsize="10%" stroke="f" fillcolor="#3AAEE0">
```

Comment:
The VML is to support desktop microsoft outlook. Button was generated from https://buttons.cm/